.changeset/odd-dragons-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@propeldata/ui-kit': patch
+---
+
+Fixed type exports
```

Comment:
This is not only type export. 

Maybe you can update it with 'Fixed component and type exports' or adding related components name.